### ISSUE TITLE
CIS: machines unreachable after they are created by terraform

### ISSUE THREAD

**Issue by aitzkowi** [CiscoCloud/mantl#1495]
I have experienced the following issue with mantl 1.1.0:
when creating resources with terraform on CIS, a subnetwork is created but DHCP is disabled.
As a result, you cannot ssh into the machines and all ansible scripts fail with "unreachable=1".
enabling DHCP on the subnetwork (via horizon UI) solves the problem.

**Comment by siddharthist** [comment]
This might be https://github.com/hashicorp/terraform/issues/6748

**Comment by siddharthist** [comment]
Actually, we manually set `enable_dhcp = true` in [our Openstack terraform configuration](https://github.com/CiscoCloud/mantl/blob/bf4327eb7e3dcc7fe107b6c3132d621335f6d1ae/terraform/openstack/network/main.tf#L19) that creates the subnet, so I'm not sure what could be going wrong, especially since the commit that added that was part of 1.1.

**Comment by aitzkowi** [comment]
have you tested on CIS specifically?

**Comment by stevendborrelli** [comment]
@siddharthist yes, that is exactly it. It looks like the next version of terraform fixes it, but we should probably enable_dhcp in our subnet module for 1.1.1.

**Comment by aitzkowi** [comment]
it occurs with terraform 0.6.16 which is the latest official version

**Comment by thomasvincent** [comment]
Is this still an issue?
